### PR TITLE
improves virology immeasurably

### PR DESCRIPTION
**Pull request by lm40**
my commits are the best, always

### CODE REVIEW

**Comment by Rykka-Stormheart** [comment]
Always the best. Singlehandedly fixes Virology, new best department 2020

**Comment by Arokha** [comment]
You're right, it is difficult to measure how much this improves virology.

File: code/modules/virus2/effect.dm [hunk 1]
```diff
@@ -136,7 +136,7 @@
 	mob.updatehealth()
 
 /datum/disease2/effect/killertoxins
-	name = "Autoimmune Reponse"
+	name = "Autoimmune Response"
```
Comment (KasparoVy) [inline code/modules/virus2/effect.dm:139]: Repose* ;)

Comment (lm40) [reply code/modules/virus2/effect.dm:139]:
```suggestion
	name = "Autoimmune Respose"
```

**Pull request merged**